lib/gtfs/components/GtfsMap.js

```diff
@@ -290,22 +305,21 @@ export default class GtfsMap extends Component<Props, State> {
           url={`https://api.tiles.mapbox.com/v4/${MAPBOX_MAP_ID}/{z}/{x}/{y}${Browser.retina ? '@2x' : ''}.png?access_token=${MAPBOX_ACCESS_TOKEN}`}
           attribution={MAPBOX_ATTRIBUTION} />
         {/* Feed version bounding box (show if not set to default indicating invalid version bounds) */}
-        {showBounds && bounds !== DEFAULT_BOUNDS &&
+        {showBounds && versionBoundingBox &&
           <Rectangle
-            bounds={bounds}
+            bounds={versionBoundingBox}
             fillOpacity={0} />
         }
 
         {/* Stops from map bounds search */}
-        {showStops && stops && stops.length && <FeatureGroup ref='stops'>
+        {(showStops || this.props.entities.indexOf('stops') !== -1) && stops && stops.length && <FeatureGroup ref='stops'>
```

Comment:
Expand this line so it's more readable.  Also, destructure the `entities` variable out of the props.